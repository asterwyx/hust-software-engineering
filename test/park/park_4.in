preset user AQ
preset userloc A 12 0
preset userloc Q 45 0
preset gift Q barrier 5
preset gift A robot 5
preset nextuser Q
step 4 #Q:45-49 park
step 2 #A:12->14 park
block 5 #Q block-
step 1
n
robot
step 1
n
dump

