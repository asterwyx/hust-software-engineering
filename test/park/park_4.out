user AQ
fund A 10000
credit A 0
gift A robot 4
userloc A 15 0
fund Q 10000
credit Q 0
gift Q barrier 4
userloc Q 50 0
nextuser Q
barrier 54